optaplanner-core/src/main/java/org/optaplanner/core/config/util/ConfigUtils.java

```diff
@@ -414,6 +414,21 @@ public static int resolvePoolSize(String propertyName, String value, String... m
         return memberAccessor;
     }
 
+    public static String abbreviate(List<String> list, int limit) {
+        String abbreviation = "";
+        if (list != null) {
```

Comment:
Nitpick:
I personally think the method should not accept null lists. Just add a javadoc that has "@param list never null"

But if should accept null too, we treat is as an exceptionally situation (so handle it with an if statement before doing the real code)
```

if (list == null) {
   return "";
}
String abbreviation = ...
```

Reply:
ah, I see below it's easier for it to also accept null.